dsrc_driver/CMakeLists.txt

```diff
@@ -108,9 +108,9 @@ find_package(Boost REQUIRED COMPONENTS system thread)
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-#  INCLUDE_DIRS include
-#  LIBRARIES pinpoint
-#  CATKIN_DEPENDS pinpoint_msgs roscpp
+  #INCLUDE_DIRS include
+  LIBRARIES dsrc_driver
```

Comment:
I don't see any libraries named dsrc_driver in this package. Seems this line is unnecessary.